app/redhat_insights/default/data/ui/views/main_dashboard.xml

```diff
@@ -55,7 +55,7 @@
 		"viz_6ySxGaeW": {
 			"type": "viz.text",
 			"options": {
-				"content": "New today",
+				"content": "New Today",
```

Comment:
Is this needed? It does not seem to be relevant.

Reply:
Thanks to notive that, this is not relevant, my main dashboard got old changes for some reason, fixed.